joblib/test/test_parallel.py

```diff
@@ -21,6 +21,7 @@
 
 from joblib.test.common import np, with_numpy
 from joblib.test.common import with_multiprocessing
+#from joblib.test.common import with_memory_profiler, memory_used
```

Comment:
Are these left-over from debug?

Reply:
Yes, we tried to see if this generator could meet the previous memory performance goal, it did not. Will clean.